### ISSUE TITLE
Auto Segmentation  - initialize threshold value by histogram of density values

### ISSUE THREAD

**Issue by amymmorton** [BrownBiomechanics/SlicerAutoscoperM#60]
In pursuit of an automated vrg generation for multiple volumes in batch  #59  .. we'd want to autorun segmentations- the hu threshold vlaue is currently static- and could be initialized by  some X% peak density value ?

**Comment by NicerNewerCar** [comment]
Definitely! This is a great idea, there are also quite a lot of ideas that I have related to improving the segmentation algorithm that I am planning on testing out once the 3D and 4D CT work gets integrated!

**Comment by amymmorton** [comment]
Also- is it possible to have the Segmentation named in association with the volume-node?

**Comment by NicerNewerCar** [comment]
Yes, this is something that I was already planning on adding because `Segmentation` and `Segmentation_1` are terrible names.

**Comment by amymmorton** [comment]
lol agreed
